cs3api4lab/api/cs3apismanager.py

```diff
@@ -25,6 +32,18 @@ def __init__(self, parent, log):
         self.log = log
         self.file_api = Cs3FileApi(self.log)
         self.share_api = ShareAPIFacade(log)
+        self._init_user_dir_setting()
+
+    def _init_user_dir_setting(self):
+        if "user_dir" in self.cs3_config and self.cs3_config["user_dir"] != "":
```

Comment:
Uhm... Not sure if I would document this instead of saving this config file.
Also, since we have a server extension that exposes an API, why don't we just create a new endpoint to return the session info with the user_dir there?

Reply:
Then we don't need to care about the config system, which we know can be modified by the user.

Reply:
@diocas This solution was based on our talk about the standard jupyter way of passing settings to the frontend, but of course we can rework it to use a simple endpoint returning the user home directory

Reply:
Yes, we discussed this. But I though, from what I understood of your explanation, that this was somehow being used by the upstream code to change the user home dir.
In the end, you added some code that `cd`'s into the directory. So, if that is the case, I think this approach brings more complexity without any advantage compared to just calling an API endpoint.